install.sh

```diff
@@ -7,9 +7,9 @@ nvm_has() {
   return $?
 }
 
-if [ -z "$NVM_DIR" ]; then
-  NVM_DIR="$HOME/.nvm"
-fi
+nvm_install_dir() {
+  echo ${NVM_DIR:-"$HOME/.nvm"}
```

Comment:
We now have our installation tests running on all supported shells so I think we're OK for now.

This could always be changed later to be an if/else.